padl/dumptools/symfinder.py

```diff
@@ -668,14 +680,65 @@ def replace_star_imports(tree: ast.Module):
                 node.names = [ast.alias(name=name, asname=None) for name in names]
 
 
+def update_scopedname(scoped_name: ScopedName, scope: Scope, add_n: int = 0, remove_dot: bool = False):
```

Comment:
You only ever use this with `add_n != 0 and remove_dot == False` or `remove_dot == True and add_n == 0`.
It would be more readable if you had two separate functions `remove_dot` and `add_n` (or something like that). (If `remove_dot` is even needed, see comment in `find_scopedname_in_source`).

Reply:
I have moved this to the methods of ScopedName.